packages/vite-node/src/cli.ts

```diff
@@ -131,6 +131,31 @@ async function run(files: string[], options: CliOptions = {}) {
     process.on('uncaughtException', (err) => {
       console.error(c.red('[vite-node] Failed to execute file: \n'), err)
     })
+
+    if (process.env.VITE_TEST_WATCHER_DEBUG) {
```

Comment:
Can we add `process.env.VITE_TEST_WATCHER_DEBUG: false` in `rollup` config (under a condition like "production" build) so this is removed when bundling?

Reply:
It looks like I can use `process.env.ROLLUP_WATCH` to remove this only on build https://github.com/vitest-dev/vitest/pull/5311/commits/17aa6951edecc12f89da0fb7ba636a77bded1ddc

Reply:
Ah, this obviously breaks CI since it uses build... Let me think about a different way.